qemu.go

```diff
@@ -280,22 +442,66 @@ func (d *Driver) Start() error {
 	// fmt.Printf("Init qemu %s\n", i.VM)
 	machineDir := filepath.Join(d.StorePath, "machines", d.GetMachineName())
 
-	startCmd := []string{
-		"-display", "none",
+	var startCmd []string
+
+	if d.Display {
+		if d.DisplayType != "" {
```

Comment:
ah - this tripped me up - I ran `machine create -d qemu --display-type vnc b2d` - and see nothing, kinda a UX pain - but can be defered :)

Reply:
```
machine create -d qemu --qemu-display-type vnc=:1 --qemu-display b2d
```

is going to be useful to me :)